CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## master / unreleased
  - [REMOVED] `chunks.NewReader` is removed as it wasn't used anywhere.
  - [REMOVED] `FromData` is considered unused so was removed.
+ - [BUGFIX] Bugfix in selectOverlappingDirs. Only return truly overlapping blocks.
```

Comment:
what `truly` actually means?

Reply:
Oh I may want to express `first`, will change later.